From 31e34b59930047c733dba55dc12ec6d9f69bea98 Mon Sep 17 00:00:00 2001
From: Jon Lin <jon.lin@rock-chips.com>
Date: Fri, 3 Jul 2020 20:02:00 +0800
Subject: [PATCH] Revert "spi: rockchip: support to set lsb into
 ROCKCHIP_SPI_CTRLR0"

This reverts commit 2c2acee8f8b5b42acbf50679d0b4f7b2e6cfb445 which is
commit 04290192f7ebe892828f69ac57c4684e25da378d.

Signed-off-by: Jon Lin <jon.lin@rock-chips.com>
Change-Id: Idda76b569aefa82b5c5d98ad5c68ec39cc8bee5d
---
 drivers/spi/spi-rockchip.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/spi/spi-rockchip.c b/drivers/spi/spi-rockchip.c
index b5f9ac0257ec..924057b72cf4 100644
--- a/drivers/spi/spi-rockchip.c
+++ b/drivers/spi/spi-rockchip.c
@@ -548,8 +548,6 @@ static void rockchip_spi_config(struct rockchip_spi *rs)
 
 	cr0 |= (rs->n_bytes << CR0_DFS_OFFSET);
 	cr0 |= ((rs->mode & 0x3) << CR0_SCPH_OFFSET);
-	if (rs->mode & SPI_LSB_FIRST)
-		cr0 |= (1 << CR0_FBM_OFFSET);/* First Bit Mode */
 	cr0 |= (rs->tmode << CR0_XFM_OFFSET);
 	cr0 |= (rs->type << CR0_FRF_OFFSET);
 
@@ -780,7 +778,7 @@ static int rockchip_spi_probe(struct platform_device *pdev)
 
 	master->auto_runtime_pm = true;
 	master->bus_num = pdev->id;
-	master->mode_bits = SPI_CPOL | SPI_CPHA | SPI_LOOP | SPI_LSB_FIRST;
+	master->mode_bits = SPI_CPOL | SPI_CPHA | SPI_LOOP;
 	master->num_chipselect = ROCKCHIP_SPI_MAX_CS_NUM;
 	master->dev.of_node = pdev->dev.of_node;
 	master->bits_per_word_mask = SPI_BPW_MASK(16) | SPI_BPW_MASK(8);
-- 
2.35.3

